wavefront-kubernetes-adapter/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: wavefront-kubernetes-adapter
```

Comment:
Let's rename the chart to `wavefront-hpa-adatper`. Would help distinguish if we ever release a VPA adapter etc in the future.